validator/validate_test.go

```diff
@@ -116,6 +116,8 @@ func runSpec(schemas []*gqlparser.Schema, deviations []*Deviation, filename stri
 					}
 				}
 
+				t.Logf("name: '%s'", spec.Name)
```

Comment:
Maybe pop this down the bottom in the `t.Failed` section with the query so the tests aren't noisy all the time?